### ISSUE TITLE
Unable to download repo using bower.json

### ISSUE THREAD

**Issue by Priyank780** [BlackrockDigital/startbootstrap-sb-admin-2#150]
Hello,

While downloading this repo using bower.json, I'm getting below error,

bower ENORESTARGET  Tag/branch master does not exist

Additional error details:
No tags found in https://github.com/IronSummitMedia/startbootstrap-sb-admin-2.git
Available branches: gh-pages

```

Is this known issue? Is there any work-around or permanent solution?

```

Here is my bower.json file,

> {
>     "name": "test",
>     "dependencies": {
>         "startbootstrap-sb-admin-2": "*",
> 
> ```
> },
> "private": true
> ```
> 
> }

**Comment by raeno** [comment]
@Priyank780 I think the problem is that you use repo that references old repository ( https://github.com/IronSummitMedia/startbootstrap-sb-admin-2.git ) instead of this one.

If you run ```bower search startbootstrap``` you will find multiple bower packages referencing this repository, for example ```blackrockdigital-startbootstrap-sb-admin-2``` or ```brd-sb-admin-2```
So the solution is to remove your current startbootstrap-sb-admin-2 from package.json and use one of packages I mentioned.

**Comment by Priyank780** [comment]
@raeno  thanks

**Comment by lwrbakro** [comment]
After `bower install brd-sb-admin-2 --save`, my bower.json looks like this:
```
"dependencies": {
    ...,
    "startbootstrap-sb-admin-2": "brd-sb-admin-2#^3.3.7+1"
  }
```

After cleaning the bower folder and reinstalling the packages with the following command: `bower install`

I get the following error msg:
```
bower ENORESTARGET  Tag/branch ^3.3.7+1 does not exist

Additional error details:
Available tags: v1.0.0, v1.0.1, v1.0.2, v1.0.3, v1.0.4, v1.0.5, v1.0.6, v1.0.7, v1.0.8, v3.3.7, v3.3.7+1
Available branches: gh-pages, master
```

**Comment by lwrbakro** [comment]
I think the format of the "v3.7.1+1" tag is not compatible with bower.
